.github/workflows/knative-mkdocs-dockerimage.yaml

```diff
@@ -0,0 +1,24 @@
+name: Docker Image CI
+
+on:
+    push:
+        branches: [ main ]
+
+jobs:
+  docker:
+    name: Build knative mkdocs dockerfile
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          file: ./hack/docker/Dockerfile
+          push: true
+          tags: krsnam/mkdocs:latest
```

Comment:
tags should point to the community's dockerhub repo instead of mine.

Reply:
knative community doesn't use dockerhub image registry

We use Prow, Prow has the credentials to push to the Knative google container registry where we have an account and store all the nightly builds.

cc @evankanderson